app/models/service_template.rb

```diff
@@ -434,6 +444,26 @@ def order(user_or_id, options = nil, request_options = nil, schedule_time = nil)
     end
   end
 
+  def update_schedule(schedule_time)
+    if schedule_time
```

Comment:
Why would schedule_time be `nil` or `false`?  Should we instead expose this through the api as separate `unschedule` (which would delete all related schedules) and `schedule` actions?

Reply:
Alberto and I discussed this and the thought was that schedules would not be exposed directly by the API but instead be exposed by the classes that support schedules.